qemu/tests/cfg/blockdev_mirror_qemuio_ready_job.cfg

```diff
@@ -0,0 +1,27 @@
+- blockdev_mirror_qemuio_ready_job:
+    only Linux
+    only filesystem iscsi_direct ceph nbd gluster_direct
+    start_vm = no
+    qemu_force_use_drive_expression = no
+    type = blockdev_mirror_qemuio_ready_job
+    virt_test_type = qemu
+    images += " data1"
+    source_images = data1
+    target_images = mirror1
+    remove_image_data1 = yes
+    force_create_image_data1 = yes
+    backup_options_data1 = sync
+    sync = full
+    storage_pools = default
+    storage_pool = default
+    qemuio_cmd = 'qemu-io -f %s %s'
```

Comment:
It's better to get qemu-io binary by utils_misc.get_qemu_io_binary(params)

Reply:
-f % -->  -f ${image_format_data1}

Reply:
Have update, thanks @zhencliu